quickstarts/redis-otel/config.yml

```diff
@@ -0,0 +1,29 @@
+slug: redis-otel
+title: Redis (OpenTelemetry)
+summary: |
+  Monitoring Redis is essential for maintaining fast database operations and a responsive application. Install New Relic Redis for OpenTelemetry quickstart to proactively instrument Redis with the OpenTelemetry Redis receiver.
+description: |
+  ## How to monitor Redis with New Relic?
+  The OpenTelemetry Redis receiver collects metrics from your running Redis instances. These metrics provide insights into operations, memory usage, client connections, and more.
+  A New Relic entity will be created for each Redis instance, allowing you to easily explore, configure alerts, and compare metrics for all your Redis databases in one place.
+  Our monitoring quickstart includes a pre-built dashboard that displays aggregated metrics from all of your Redis instances, with the ability to filter the data for specific analyses.
+  ### Why monitor Redis with New Relic?
+  New Relic Redis for OpenTelemetry quickstart gives you comprehensive visibility into your Redis databases alongside your apps and server infrastructure. Monitoring Redis is vital to obtain real-time performance metrics for databases across your entire environment. This ensures optimal database operations, leading to better application performance and enhanced user experiences.
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+dataSourceIds:
+  - redis-otel
+alertPolicies:
+  - redis-otel
+documentation:
+  - name: Redis (OpenTelemetry) installation docs
+    description: |      
+      Redis is an in-memory data structure store, used as a database, cache, and message broker. Monitoring it provides insights into its operations and performance.
+    url: https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-redis
```

Comment:
This link does not seem to exist on our documentation site.

Reply:
We're waiting for this PR to be ready to be merged before releasing docs.
Docs are referring to this Quickstart so we should release docs and quickstarts at the same time.